app/src/main/java/com/jerboa/ui/components/post/edit/PostEditScreen.kt

```diff
@@ -131,6 +128,22 @@ fun PostEditScreen(
                     }
                 },
                 isUploadingImage = isUploadingImage,
+                altText = altText,
+                onAltTextChange = { altText = it },
+                customThumbnailField = customThumbnailField,
+                customThumbnail = customThumbnail,
+                onCustomThumbnailChange = { customThumbnail = it },
+                isUploadingCustomThumbnailImage = isUploadingCustomThumbnailImage,
+                onCustomThumbnailImagePicked = { uri ->
+                    if (!account.isAnon() && uri != Uri.EMPTY) {
+                        val imageIs = imageInputStreamFromUri(ctx, uri)
```

Comment:
I see that we do this in a lot of places but does this actually work on API level 26-28 ?